tests/internal/service_name/test_imports.py

```diff
@@ -91,7 +89,7 @@ def test_service_name_import_with_client_service_names_enabled_v0():
     from ddtrace.internal.schema.span_attribute_schema import service_name_v1
     from ddtrace.internal.schema.span_attribute_schema import url_operation_v0
 
-    assert DEFAULT_SPAN_SERVICE_NAME == DEFAULT_SERVICE_NAME
+    assert DEFAULT_SPAN_SERVICE_NAME == "ddtrace_subprocess_dir"
```

Comment:
do we want to have a constant for this? seems to be repeated a bunch.

Reply:
added one.